development/docker-compose-test-scale.yml

```diff
@@ -1,138 +0,0 @@
----
```

Comment:
not sure if we still need this file or not @fatih-acar do you know ?

Reply:
That's used by the scale testing CI job but it's been broken for a while now...
We probably want to move that job in a private repo anyway and fix it there?